test/turbopack-tests-manifest.json

```diff
@@ -4671,13 +4672,6 @@
     "flakey": [],
     "runtimeError": false
   },
-  "test/e2e/app-dir/scss/multi-global-reversed/multi-global-reversed.test.ts": {
-    "passed": ["Multi Global Support (reversed) should render the page"],
-    "failed": [],
-    "pending": [],
-    "flakey": [],
-    "runtimeError": false
-  },
```

Comment:
Why does this disappear? @kwonoj @timneutkens

Reply:
Can't say for sure, but looks like some name collision happened while generating manifest.

created https://github.com/vercel/next.js/pull/63060 to confirm.

Reply:
I believe this is resolved by https://github.com/vercel/next.js/pull/63066